doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -160,31 +165,67 @@ The following list summarizes the most important changes inherited from upstream
 Zephyr
 ======
 
-.. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``730acbd6ed`` (``v2.6.0-rc1``), plus some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``14f09a3b00``, plus some |NCS| specific additions.
 
 For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
 
 .. code-block:: none
 
-   git log --oneline v2.6.0-rc1 ^v2.4.99-ncs1
+   git log --oneline 14f09a3b00 ^v2.6.0-rc1-ncs1
 
 For a complete list of |NCS| specific commits, run:
 
 .. code-block:: none
 
-   git log --oneline manifest-rev ^v2.6.0-rc1
+   git log --oneline manifest-rev ^14f09a3b00
 
-The current |NCS| release is based on Zephyr v2.6.0-rc1.
-See the :ref:`zephyr:zephyr_2.6` Release Notes for an overview of the most important changes inherited from upstream Zephyr.
+The current |NCS| master branch is based on the Zephyr v2.7 development branch.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-* The ``nrf9160dk_nrf9160ns`` and the ``nrf5340dk_nrf5340_cpuappns`` boards have been renamed respectively to ``nrf9160dk_nrf9160_ns`` and ``nrf5340dk_nrf5340_cpuapp_ns``.
-* A config option for ``memcpy`` that skips the word-based loop before the byte-based loop was added.
-  It is now enabled by default if :kconfig:`SIZE_OPTIMIZATIONS` is set.
-  As result, any application-specific assumptions about ``memcpy`` read or write size behavior should be rechecked if this option is enabled.
+.. TODO update the following sections to reflect https://github.com/nrfconnect/sdk-nrf/pull/5189. Delete this comment once that is complete.
```

Comment:
@mbolivar-nordic , correct me if I'm wrong, but I remember a conversation with @carlescufi and @shantha-14 around the NCS 1.6.0 release where it was decided that we would only link to the Zephyr release notes and skip listing all the changes. So these sections should be removed and we should just have a link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/releases/release-notes-2.7.html , with potentially a subsection detailing more complex changes, like in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-1.6.0.html#zephyr : 
```suggestion
See the :ref:`zephyr:zephyr_2.7` Release Notes for an overview of the most important changes inherited from upstream Zephyr up to the commit ``14f09a3b00``.
```